src/HMI/lux.js

```diff
@@ -1498,7 +1498,7 @@ LUX.HMI = function (...dataCallbacks) {
 			
 			// add variable if it doesn't exist yet
 			if ( this.value.call(this, variableName) === undefined) {
-				this.writeVariable(variableName, 'undefined');
+				this.writeVariable(variableName, '');
```

Comment:
Why are we doing this at all?

Reply:
This is for local data, not data written to the plc. We implicitly create read variables for local data